thirdparty/google-styleguide/cppguide.html

```diff
@@ -0,0 +1,6170 @@
+<!DOCTYPE html>
```

Comment:
Should likely just link to this in docs some place.  As best as I can tell, this file is CC-By 3.0 as per https://github.com/google/styleguide/blob/gh-pages/README.md.  This falls under a Category B (http://www.apache.org/legal/resolved.html#category-b) which seems to have some caveats best avoided unless there is a compelling reason to include